org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/utils/XMLPositionUtility.java

```diff
@@ -247,36 +247,57 @@ static DOMNode findUnclosedChildNode(String childTag, List<DOMNode> children) {
 
 	public static Range selectRootStartTag(DOMDocument document) {
 		DOMNode root = document.getDocumentElement();
-		return selectStartTag(root);
+		return selectStartTagName(root);
 	}
 
-	public static Range selectStartTag(int offset, DOMDocument document) {
+	public static Range selectStartTagName(int offset, DOMDocument document) {
 		DOMNode element = document.findNodeAt(offset);
 		if (element != null) {
-			return selectStartTag(element);
+			return selectStartTagName(element);
 		}
 		return null;
 	}
 
 	/**
-	 * Returns the range of the start tag of the given <code>element</code> and null
+	 * Returns the range of the start tag name (excludes the '<') of the given <code>element</code> and null
 	 * otherwise.
 	 * 
 	 * @param element the DOM element
 	 * @return the range of the start tag of the given <code>element</code> and null
 	 *         otherwise.
 	 */
-	public static Range selectStartTag(DOMNode element) {
-		int startOffset = element.getStart() + 1; // <
-		int endOffset = startOffset + getStartTagLength(element);
+	public static Range selectStartTagName(DOMNode element) {
+		return selectStartTagName(element, false);
+	}
+
+	/**
+	 * Returns the range of the start tag name (excludes the '<') of the given <code>element</code> and null
+	 * otherwise.
+	 * 
+	 * @param element the DOM element
+	 * @param suffixOnly select the suffix portion, only when a prefix exists
+	 * @return the range of the start tag of the given <code>element</code> and null
+	 *         otherwise.
+	 */
+	public static Range selectStartTagName(DOMNode element, boolean suffixOnly) {

```

Comment:
* rename suffixOnly to localNameOnly
 * Set this method as private and create a new method selectStartLocalName which call selectStartTageName (element, true)